### ISSUE TITLE
style overriding default android header bar alignment

### ISSUE THREAD

**Issue by KholofeloMoyaba** [rajeshwarpatlolla/ionic-datepicker#178]
One of the styles added by this component inside `ionic-datepicker/src/ionic-datepicker.styles.css` is as follows:

```
.platform-android .bar .title.title-left {
  text-align: center;
}
```

This is a problem because it is a global style and as such it changes the app's look. I was surprised when my android app suddenly had misaligned titles.

please either remove this style or make it targeted by adding an extra class i.e. `.platform-android .ionic_datepicker_modal .bar .title.title-left`

**Comment by rajeshwarpatlolla** [comment]
Thanks for finding this issue @KholofeloMoyaba, I will modify that css with your suggestion. You can expect this change in the next release.

**Comment by rajeshwarpatlolla** [comment]
This has been fixed in v1.1.0
